libs/auth-api-lib/src/lib/delegations/delegations-index.service.ts

```diff
@@ -24,6 +29,18 @@ type SortedDelegations = {
   deleted: DelegationIndexInfo[]
 }
 
+const hasAllSameScopes = (a: string[] | undefined, b: string[] | undefined) => {
+  // Only custom delegations have scopes and they are never undefined
+  if (!a && !b) {
+    return true
+  }
+
+  if (a?.length !== b?.length) {
+    return false
+  }
+  return a?.every((s) => b?.includes(s))
```

Comment:
`=== true` so it doesn't return undefined in any case?

Reply:
Will fix 👍